support/cas-server-support-oauth-core-api/src/main/java/org/apereo/cas/support/oauth/authenticator/OAuth20ClientIdClientSecretAuthenticator.java

```diff
@@ -29,12 +30,34 @@
  * @since 5.0.0
  */
 @Slf4j
-@RequiredArgsConstructor
 public class OAuth20ClientIdClientSecretAuthenticator implements Authenticator<UsernamePasswordCredentials> {
+
+    /**
+     * {@link TicketRegistry} for storing and retrieving tickets as needed.
+     */
+    protected final TicketRegistry ticketRegistry;
+
+    /**
+     * The ticket cipher, if any.
+     */
+    protected final CipherExecutor<Serializable, String> registeredServiceCipherExecutor;
```

Comment:
Make all private, unless there is a need to open up the access control?

Reply:
This was required in `OAuth20ProofKeyCodeExchangeAuthenticator`, so opened up access to `protected`